libraries/node-core-library/src/FileWriter.ts

```diff
@@ -33,6 +34,15 @@ export interface IFileWriterFlags {
   exclusive?: boolean;
 }
 
+/**
+ * An alias for the Node.js `fs.Stats` object.
+ *
+ * @remarks
+ * This avoids the need to import the `fs` package when using the {@link FileWriter} API.
+ * @public
+ */
+export type FileWriterStats = fs.Stats;
```

Comment:
Just grab `FileSystemStats` from `FileSystem.ts`

Reply:
You mean removing type alias completely or defining it like `type FileWriterStats = FileSystemStats`?

Reply:
I’ve removed the FileWriterStats type alias in favour of FileSystemStats